src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -694,6 +652,27 @@ final case class ProcessApiServiceImpl(
       suspendDescriptorResponse[Unit](operationLabel)(_ => suspendDescriptor204)
     }
   }
+
+  override def getEServiceDocumentById(eServiceId: String, descriptorId: String, documentId: String)(implicit
```

Comment:
Move this code back to where it was please! 👍🏼

Reply:
+1. Don't move code around if not strictly necessary, it makes the PR way more difficult to be reviewed